content/docs/studio/troubleshooting.md

```diff
@@ -88,3 +96,48 @@ metrics or hyperparameters that you want to visualize.
 Refer to the [DVC documentation](https://dvc.org/doc) for help on making commits
 to a DVC repository. Instructions on how to specify custom files can be found
 [here](/doc/studio/user-guide/views/view-settings#configuring-view-settings).
+
+## View does not contain the columns that I want
+
+There are two possible reasons for this:
+
+1. **The required columns were not imported:** DVC Studio will import up to 200
+   columns (metrics, hyperparameters and files) from your Git repository. If
+   your repository has more than 200 columns, you should
+   [select the columns that are mandatory to import](/doc/studio/user-guide/views/view-settings#mandatory-columns).
+   DVC Studio will also import the unselected columns, but only up to a maximum
+   of 200 columns.
+
+   **What if there are more than 200 mandatory columns?**<br>Currently DVC
+   Studio cannot import over 200 columns. If you have a large repository (with
+   more than 200 mandatory columns), one solution is to create a mono repo - if
+   your ML project can be divided into sub-projects, create a
+   [monorepo](https://en.wikipedia.org/wiki/Monorepo) where each sub-project
+   resides in a separate directory in the monorepo. Then, for each sub-project,
+   create a view.<br><br>To create views in the case of a monorepo,
+   [specify the project directory in view settings](/doc/studio/user-guide/views/view-settings#project-directory).<br><br>If
+   this solution does not work for your use case, please create a support ticket
+   in the
+   [DVC Studio support GitHub repository](https://github.com/iterative/studio-support).
+
+2. **The required columns are hidden:** In the view, you can hide the columns
+   that you do not want to display. If any column that you want is not visible,
+   make sure you have not hidden it. The following video shows how you can
+   show/hide columns. Once you show/hide columns, remember to save the changes.
+
+   ###### Show/hide columns
+
+   ![](https://static.iterative.ai/img/studio/show_hide_columns.gif)
```

Comment:
Is this an H6? Probably better to use regular bold (`**`), or an image cation:

```suggestion
   ![](https://static.iterative.ai/img/studio/show_hide_columns.gif) _Show/hide columns_
```

Reply:
I actually don't want that header at all. But I couldn't figure out how else to create an anchor which I can link to from other places in the doc. Any suggestions?

Reply:
Have added alt text.